src/main/java/beam/analysis/plots/PersonVehicleTransitionStats.java

```diff
@@ -32,8 +33,13 @@
     private static Map<String, TreeMap<Integer, Integer>> onRoutes = new HashMap();
     private static Map<String, Integer> modePerson = new HashMap<>();
     private static final String fileName = "tripHistogram";
-    private final int binSize = 300;
-    private final int nofBins = 30 * 3600 / binSize + 1;
+    private  int binSize = 0;
+    private  int nofBins = 0;
```

Comment:
Suggestions:
- Fix the formatting: there are two spaces between `private` and `int`. 
- There is no need to initialize with default value.
- Make the name of the variable `nofBins` more meaningful.

```Java
private int binSize;
private int numOfBins;
```